src/main/scala/cromwell/engine/callactor/BackendCallActor.scala

```diff
@@ -0,0 +1,19 @@
+package cromwell.engine.callactor
+
+import cromwell.engine.callexecution.CallExecutionActor
+import cromwell.engine.{AbortRegistrationFunction, WorkflowDescriptor}
+import cromwell.engine.backend.Backend
+import cromwell.engine.workflow.BackendCallKey
+import wdl4s._
+
+class BackendCallActor(override val key: BackendCallKey,
+                       locallyQualifiedInputs: CallInputs,
+                       backend: Backend,
+                       override val workflowDescriptor: WorkflowDescriptor) extends CallActor {
+
+  override def getCallExecutionActor = {
```

Comment:
not to mention using `get` as in `getFoo` is frowned in scala.

Actually `get` as in `foo.get` is as well ;)